src/client/datascience/commands/commandRegistry.ts

```diff
@@ -477,14 +482,18 @@ export class CommandRegistry implements IDisposable {
         }
     }
 
-    private async createNewNotebook(): Promise<void> {
+    private async createNewNotebook(): Promise<INotebookEditor | undefined> {
         if (this.useNativeNotebook) {
-            await this.nativeNotebookCreator.createNewNotebook();
+            return await this.nativeNotebookCreator.createNewNotebook();
```

Comment:
Isn't return await redundant? Thought there was an eslint rule for that.

Reply:
Yep, it is and removed all instances of it. Ty